source/file_management/file_management.rst

```diff
@@ -222,19 +221,38 @@ and ``build_systems``, for project-specific build systems.
     }
 
 
-**Folders**
```

Comment:
I think it makes sense to introduce a reference doc for sublime-project files, no? I mean, it's a file format too.